Resources/config/requirejs.yml

```diff
@@ -20,6 +21,7 @@ config:
                         urls:
                             list: pim_customentity_rest_list
                             get: pim_customentity_rest_get
+                            getall: pim_customentity_rest_get
```

Comment:
Are you sure of this route?
It's the same than `get`

Reply:
I confirm the error, but it works by chance because we don't provide the id in the fetcher :wink:

Reply:
yeah, good catch. I'll fix that up